specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.0/examples/SuccessfulEntityLinkingRequest.json

```diff
@@ -0,0 +1,99 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "Endpoint": "{Endpoint}",
+    "input": {
+      "documents": [
+        {
+          "language": "en",
+          "id": "1",
+          "text": "I had a wonderful trip to Seattle last week."
+        },
+        {
+          "language": "en",
+          "id": "2",
+          "text": "I work at Microsoft."
+        },
+        {
+          "language": "en",
+          "id": "3",
+          "text": "I visited Space Needle 2 times."
+        }
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "documents": [
+          {
+            "id": "1",
+            "entities": [
+              {
+                "name": "Seattle",
+                "matches": [
+                  {
+                    "text": "Seattle",
+                    "offset": 26,
+                    "length": 7,
+                    "score": 0.15046201222847677
+                  }
+                ],
+                "language": "en",
+                "id": "Seattle",
+                "url": "https://en.wikipedia.org/wiki/Seattle",
+                "dataSource": "Wikipedia"
+              }
+            ],
+            "warnings": []
+          },
+          {
+            "id": "2",
+            "entities": [
+              {
+                "name": "Microsoft",
+                "matches": [
+                  {
+                    "text": "Microsoft",
+                    "offset": 10,
+                    "length": 9,
+                    "score": 0.1869365971673207
```

Comment:
Same as previous comments in regards to confidence scores and 2 decimal precision.

Reply:
Changed